### PR TITLE
Feat(stylelint-config): Allow private SASS variables prefixed with `_`

### PR DESCRIPTION
**Pull request by vmilersky**
+ Provide rule name at the end of custom stylelint rule message.

### CODE REVIEW

**Review by adamkudrna — APPROVED** [review]

**Comment by adamkudrna** [comment]
Jist one thing: it's a BC because you are disallowing private variables prefixed with `-`.

File: packages/stylelint-config/rules/extras.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ module.exports = {
     'selector-nested-pattern': [
       '(^&:)|(^&\\[)|(^&\\.(is\\-|has\\-))',
       {
-        message: 'Only pseudo class selector, attribute selector and combination with state class is allowed',
+        message:
+          'Only pseudo class selector, attribute selector and combination ' +
```
Comment (literat) [inline packages/stylelint-config/rules/extras.js:15]: What about the usage of template literals? They are not allowed or do you see a different problem?

**Review by literat — APPROVED** [review]
LGTM

File: packages/stylelint-config/rules/extras.js (hunk 1 above)
Comment (vmilersky) [reply packages/stylelint-config/rules/extras.js]: No need for using literals. This string was split by mistake. Thought that linter doesn't like the lenght of the line but it was just a code style issue that the value of the message was previously not at its own line.

**Pull request merged**